### DISCUSSION TITLE
Should we ask about early stage Open UI stuff?

### DISCUSSION THREAD

**Discussion by LeaVerou** [Devographics/surveys#117] [State of HTML Design]
E.g. things like [`<breadcrumb>`](https://github.com/Devographics/surveys/discussions/116)

**Comment by captainbrosset** [comment]
+1 to asking about early stage Open UI things. Open UI is filling a pretty important gap on the web, I think it makes sense to keep asking web devs whether they need more control over their web UI. Ideally, if we keep running this year over year, we'd see a trend. Or we can identify unmet needs.

**Comment by hidde** [comment]
Agreed with @captainbrosset 's comment that Open UI can fill an important gap, but I feel the “early stage” things like breadcrumb may be too early, as they don't seem to be in current development. My worry is that they are too hypothetical at this point.

Maybe it would be better to focus on somewhat-advanced stage Open UI stuff; I don't mean just things already shipping (like `popover`), but items that come back regularly in telecons like exclusive accordions and selectmenu. 

In addition, there could be more general questions around the gap-filling, eg what sort of gaps do developers find most urgent for their work, or what kind of things do they wish never to do by hand / have the browser do for them?

**Comment by mfreed7** [comment]
> Agreed with @captainbrosset 's comment that Open UI can fill an important gap, but I feel the “early stage” things like breadcrumb may be too early, as they don't seem to be in current development. My worry is that they are too hypothetical at this point.
> 
> Maybe it would be better to focus on somewhat-advanced stage Open UI stuff; I don't mean just things already shipping (like `popover`), but items that come back regularly in telecons like exclusive accordions and selectmenu.
> 
> In addition, there could be more general questions around the gap-filling, eg what sort of gaps do developers find most urgent for their work, or what kind of things do they wish never to do by hand / have the browser do for them?

+1 to this entire comment. I think it makes sense to ask the general "what's your top missing feature" type question to see where the gaps are. But trying to dig into anything in detail too early (e.g. `<breadcrumb>`) is probably not very efficient. I'd love to see us dig into some of the later stage things from OpenUI like popover=hint, hover-triggered popovers, exclusive accordions, improved details/summary a11y and styling, and `<selectmenu>`.
